doc/modules/ROOT/pages/tutorials/visualize-with-pyvis.adoc

```diff
@@ -0,0 +1,201 @@
+subgraph, and choose which start node around which the subgraph will be
+sampled.
+
+[source, python, role=no-test]
+----
+G_sample, _ = gds.alpha.graph.sample.rwr("cora_sample", G, randomSeed=42, concurrency=1)
+----
+
+We should have somewhere around 0.15 * 2708 ~ 406 nodes in our sample.
+And let’s see how many relationships we got.
+
+[source, python, role=no-test]
+----
+print(f"Number of nodes in our sample: {G_sample.node_count()}")
+print(f"Number of relationships in our sample: {G_sample.relationship_count()}")
+----
+
+Let’s also compute
+https://neo4j.com/docs/graph-data-science/current/algorithms/page-rank/[PageRank]
+on our sample graph, in order to get an importance score that we call
+``rank'' for each node. It will be interesting for context when we
+visualize the graph.
+
+[source, python, role=no-test]
+----
+gds.pageRank.mutate(G_sample, mutateProperty="rank")
+----
+
+== Exporting the sampled Cora graph
+
+We can now export the topology and node properties of our sampled graph
+that we want to visualize.
+
+Let’s start by fetching the relationships.
+
+[source, python, role=no-test]
+----
+sample_topology_df = gds.beta.graph.relationships.stream(G_sample)
+display(sample_topology_df)
+----
+
+We get the right amount of rows, one for each expected relationship. So
+that looks good.
+
+Next we should fetch the node properties we are interested in. Each node
+will have a ``subject'' property which will be an integer 0,…,6 that
+indicates which of seven academic subjects the paper represented by the
+nodes belong to. We will also fetch the PageRank property ``rank'' that
+we computed above.
+
+[source, python, role=no-test]
+----
+sample_node_properties_df = gds.graph.nodeProperties.stream(
+    G_sample,
+    ["subject", "rank"],
+    separate_property_columns=True,
+)
+display(sample_node_properties_df)
+----
+
+Now that we have all the data we want to visualize, we can create a
+network with PyVis. We color each node according to its ``subject'', and
+size it according to its ``rank''.
+
+[source, python, role=no-test]
+----
+net = Network(notebook = True,
+cdn_resources="remote",
+                bgcolor = "#222222",
+                font_color = "white",
+                height = "750px", # Modify according to your screen size
+                width = "100%",
+)
+
+# Seven suitable light colors, one for each "subject"
+subject_to_color = ["#80cce9", "#fbd266", "#a9eebc", "#e53145", "#d2a6e2", "#f3f3f3", "#ff91af"]
+
+# Add all the nodes
+for _, node in sample_node_properties_df.iterrows():
+    net.add_node(int(node["nodeId"]), color=subject_to_color[int(node["subject"])], value=node["rank"])
+
+# Add all the relationships
+net.add_edges(zip(sample_topology_df["sourceNodeId"], sample_topology_df["targetNodeId"]))
+
+net.show("cora-sample.html")
+----
+
+Unsuprisingly we can see that papers largely seem clustered by academic
+subject. We also note that some nodes appear larger in size, indicating
```

Comment:
just the visualization does not explain to us what the academic subjects are right?
How about we add the subject as a label?